### PR TITLE
[cordova] Add tests for googlemaps plugin

### PR DESCRIPTION
**Pull request by haoxli**
- Set z order of web view with the preference xwalkZOrderOnTop
- Cordova GoogleMaps plugin for Android
- Fail Analyse: <br/>
   XWALK-5869 xwalkZorderOnTop does not make webview in top level.<br/>
  XWALK-5868 ANIMATABLE_XWALK_VIEW is not false by default

Impacted TCs num(approved): New 1, Update 0, Delete 0
Unit test platform: cordova-plugin-crosswalk-webview
Android test result summary: Pass 0, Fail 1, Block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-5779

### CODE REVIEW

**Comment by haoxli** [comment]
@zhuyongyong @Honry Please review. The index.html need to add copyright and license.

File: cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    os.chdir(BUILD_TMP)
+    build_src = BUILD_TMP +"/HelloMap"
+    if os.path.exists(build_src):
+        shutil.rmtree(build_src)
+
+    os.system('cordova create HelloMap com.example.hellomap HelloMap')
+    os.chdir(build_src)
+    os.system('cordova platform add android')
+    os.system('cordova plugin add https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview')
```
Comment (Honry) [inline cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py]: Please use local downloaded cordova-plugin-crosswalk-webview.

File: cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    os.system('cordova plugin add https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview --variable API_KEY_FOR_ANDROID="%s"' % key)
```
Comment (Honry) [inline cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py]: This should be the google map repo.

File: cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    shutil.copyfile(SCRIPT_DIR + '/index.html', build_src  + '/www/index.html')
+    # Update android:theme="@android:style/Theme.Black.NoTitleBar" to android:theme="@android:style/Theme.Translucent.NoTitleBar" in AndroidManifest.xml
+    os.system('sed -i "s/%s/%s/g" %s' % ("@android:style\/Theme.Black.NoTitleBar", "@android:style\/Theme.Translucent.NoTitleBar", build_src + "/platforms/android/AndroidManifest.xml"))
+    # Set zOrderOnTop in config.xml
+    lines = open(build_src + '/config.xml', 'r').readlines()
+    lines.insert(-1, '    <preference name="zOrderOnTop" value="true" />\n')
```
Comment (Honry) [inline cordova/cordova-sampleapp-android-tests/sampleapp/res/GoogleMapsPlugin/google-maps-plugin.py]: This config can be set via command line.

**Comment by Honry** [comment]
LGTM

**Pull request merged**